hibernate-core/src/test/java/org/hibernate/test/type/contributor/ArrayType.java

```diff
@@ -33,4 +43,7 @@ public String getName() {
         return "comma-separated-array";
     }
 
+    public Map<String, Object> getSettings() {
```

Comment:
Not a fan of exposing these.  The whole point of the issue/PR is to allow Types to be settings-aware, not make the consumer of a Type settings-aware.  You expose it for a test, but I can guarantee with a fairly high-degree of certainty that someone will try to use this.  And at that consumer needs to protect for null every time (here for example with the static ref or non-parameterized ctor)

Reply:
Sorry, just realized this is a ArrayType from the tests, not the "main" one.